addon/index.js

```diff
@@ -72,6 +72,8 @@ export class QueryParamAwarePretender extends Pretender {
       pathnameHandlersMap.set(pathname, handler);
     }
 
+    // sort query params so that the order doesn't matter
+    search = normalizeQueryString(url);
```

Comment:
This should be optional - there's no reason to make this lib not work in cases where query param order matters, but I certainly agree such cases are rare and the dx should be optimized for the common case.

I'd suggest adding an option on `QueryParamAwarePretender` that defaults to normalizing URLs.


```
new QueryParamAwarePretender({
  // default to true
  normalizeURLs: false
});
```

What do you think?

Reply:
That's a great suggestion, I have added a flag but I think my constructor looks wonky. Any better way to check if a value is `false` ?
Also, I wanted to highlight that the parent constructor (PretenderJS) accepts either an object or a method. Hope that's not a problem since I am calling `super(arguments)`

Reply:
the typechecking you did is 👍